### PR TITLE
docs(lettable-operators): example of own operators

### PR DESCRIPTION
**Pull request by JLarky**

### CODE REVIEW

File: doc/lettable-operators.md [hunk 1]
```diff
@@ -95,11 +95,13 @@ const takeEveryNth = (n: number) => <T>(source: Observable<T>) =>
...
   });
 
+const takeEveryNthSimple = (n: number) => <T>(source: Observable<T>) =>
+  source.pipe(filter((value, index) => index % n === 0 ))
 
 interval(1000).pipe(
   takeEveryNth(2),
   map(x => x + x),
-  takeEveryNth(3),
+  takeEveryNthWithPipe(3),
```
Comment (rgbkrk) [inline doc/lettable-operators.md]: Did you mean to make this `takeEveryNthSimple`

Comment (JLarky) [reply doc/lettable-operators.md]: yes, I decided to rename it to Simple but forgot this place :)

**Comment by JLarky** [comment]
@rgbkrk pushed amended version with takeEveryNthSimple in both places

**Comment by JLarky** [comment]
Is there anything else I can do to increase chances of PR being merged? :)

File: doc/lettable-operators.md [hunk 2]
```diff
@@ -95,11 +95,13 @@ const takeEveryNth = (n: number) => <T>(source: Observable<T>) =>
     })
   });
 
+const takeEveryNthSimple = (n: number) => <T>(source: Observable<T>) =>
```
Comment (benlesh) [inline doc/lettable-operators.md]: Interestingly, you could simplify this further like so:

```ts
const takeEveryNthSimplest = (n: number) => filter((_, index) => index % n === 0);
```

... the primary point here was to show the deepest example in the simplest way... But I suppose we can go into more detail.

Comment (benlesh) [reply doc/lettable-operators.md]: LGTM

**Comment by benlesh** [comment]
Thanks @JLarky ... sorry for the slow review. I've been busy. ;)

**Pull request merged**